docs/recipes/gulp/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const gulp = require('gulp');
+const connect = require('gulp-connect');
+const lighthouse = require('../../../lighthouse-core');
+const perfConfig = require('../../../lighthouse-core/config/perf.json');
+const port = 8080;
+
+const connectServer = function() {
+  return connect.server({
+    root: '../public',
+    livereload: true,
+    port: port
+  });
+};
+
+function handleOk() {
+  connect.serverClose();
+  process.exit(0);
```

Comment:
I wonder if we can add something here to check the score and pass/fail based on that? This will always exit cleanly if running LH succeeds.